cmd/debos/debos.go

```diff
@@ -200,8 +207,11 @@ func main() {
 
 	if !fakemachine.InMachine() {
 		for _, a := range r.Actions {
+			// Stack PostCleanup methods
+			defer a.PostCleanup(&context)
```

Comment:
I find that using defer here makes it very hard to reason about when the actions are ran. I'd much rather see this be done explicitly.

Reply:
Looking at it some more; I do find it awkward but not using defers would mean more refactoring then is worth atm, so ignore those commetns for now